scripts/gke/deployment_manager_configs/iam_bindings_template.yaml

```diff
@@ -0,0 +1,40 @@
+# This config is used by iam_patch.py. It is not a DM config.
+bindings:
+- members:
+  - set-kubeflow-admin-service-account
+  roles:
+  # Grant permissions needed to push the app to a cloud repository
+  - roles/source.admin
```

Comment:
Can we follow binding schema defined here: https://cloud.google.com/iam/reference/rest/v1/Policy?
Each binding has one role and a list of members.